resources/views/myhub/company/details.blade.php

```diff
@@ -25,7 +25,7 @@
                         @livewire('company.manage-logo', ['company' => $company])
                         <x-section-border/>
                         @livewire('company.details-form', ['company' => $company ])
```

Comment:
The same case with the editing of company details

<img width="629" alt="image" src="https://github.com/HZ-HBO-ICT/it-conference/assets/77020646/af317d02-2e16-41d6-abdc-ab87e8198b93">

Reply:
Fixed in [21cc775](https://github.com/HZ-HBO-ICT/it-conference/pull/411/commits/21cc7752c176550b619e7452da865ba6368d5446)